### ISSUE TITLE
Review ActiveX related patch

### ISSUE THREAD

**Issue by lextm** [dockpanelsuite/dockpanelsuite#401]
https://github.com/transistor1/dockpanelsuite/commit/adcbf05a1483e926dc97aeb9f72f1b989e697ad9

Check if it helps.

**Comment by lextm** [comment]
Cherry picked as e62de372ddba420e1e82581853fe9584d3aad438

**Comment by billybednar** [comment]
Anyone know more about the reasoning behind this patch?

**Comment by lextm** [comment]
You might ping @transistor1 for more details.

**Comment by transistor1** [comment]
@billybednar, Please see https://github.com/transistor1/dockpanelsuite/issues/1

I'm using DockPanelSuite in [Access SQL Editor](http://fieldeffect.info/wp/access-sql-editor-documentation/), which is an ActiveX object.  PreFilterMessage doesn't get called, because it's not a normal window.

I'm curious to know - how does this relate to the bug you're tracking down?

**Comment by lextm** [comment]
@transistor1 there is an OpenGL related issue #304. I thought it might be related to this, but it seems to be irrelevant. If you also master OpenGL, you can shed some light too. Appreciate it.

**Comment by billybednar** [comment]
@transistor1 See the two issues I just opened

**Comment by transistor1** [comment]
I fixed an intermittent crash in my own fork a while back:. https://github.com/transistor1/dockpanelsuite/issues/4

Your case #456 sounds similar but I haven't reviewed it.

I forked DockPanelSuite because I have a special use case & don't expect that many users would need to host DPS in an ActiveX control.  I didn't expect to see my changes make it to the master.  @lextm if #401 needs to be backed out, I'm fine with that.
